pyproject.toml

```diff
@@ -23,10 +23,7 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
 ]
 dependencies = [
-    # install `psycopg2` on linux (assumed production)
-    'psycopg2>=2.9,<3.0; platform_system == "Linux"',
-    # install `psycopg2-binary` on macos/windows (assumed development)
-    'psycopg2-binary>=2.9,<3.0; platform_system != "Linux"',
+    "psycopg2-binary>=2.9,<3.0",
```

Comment:
It's nice this turned out so easy -- or at least it seems that way? Any major issues or were tests happy to run on the binary package?

Reply:
The packages are identical. The source code is the same repo (e.g. try going to `psycopg2-binary` on PyPI and click through to the documented source). The only difference is whether you get pre-built wheels or if you build from source on your local machine. Put another way, if doing this broke `dbt-postgres`, that would be a bug in `psycopg2`.

That being said, that means we need to update Cloud to implement the workaround, otherwise we'll use `psycopg2-binary`, which is slower than `psycopg2`. I still need to spin up that PR, which is a prerequisite to releasing this PR.